Makefile

```diff
@@ -1,23 +1,23 @@
-validate-dydx-testnet-3-gentx:
+validate-dydx-testnet-4-gentx:
 	docker build --platform linux/amd64 --tag public-testnet-validate $(CURDIR)
 	docker run --platform linux/amd64 \
-		-e VALIDATE_GENESIS_DIR_PATH=./dydx-testnet-3 \
+		-e VALIDATE_GENESIS_DIR_PATH=./dydx-testnet-4 \
 		-e VALIDATE_GENESIS_TAR_URL='https://github.com/dydxprotocol/v4-chain/releases/download/protocol%2Fv0.2.1/dydxprotocold-v0.2.1-linux-amd64.tar.gz' \
```

Comment:
Let's point this to the latest release `v0.3.6`

Reply:
roger that